docs/ABOUT.md

```diff
@@ -0,0 +1,5 @@
+Rust is a systems programming language focused on three goals: safety, speed, and concurrency. Rust does not use a garbage collector, but instead ensures safety and optimizes code at compile time. The concept of _ownership_ is how Rust achieves its largest goal, memory safety.
```

Comment:
Is there a good blog post or something about ownership? I can't see what it might mean without a bit of extra context.